cmd/arena/commands/top_node.go

```diff
@@ -151,23 +175,56 @@ func displayTopNodeSummary(nodeInfos []NodeInfo) {
 		strconv.FormatInt(allocatedGPUsInCluster, 10),
 		strconv.FormatInt(totalGPUsInCluster, 10),
 		int64(gpuUsage))
-	// fmt.Fprintf(w, "%s\t%s\t%s\t%s\t%s\n", ...)
+
+	fmt.Fprintf(w, "Used/Total CPUs In Cluster:\n")
+	log.Debugf("cpu: %s, used CPUs %s", resource.NewMilliQuantity(totalCPUsInCluster, resource.DecimalSI).String(),
+		resource.NewMilliQuantity(usedCPUsInCluster, resource.DecimalSI).String())
+	var cpuUsage float64 = 0
+	if totalCPUsInCluster > 0 {
+		cpuUsage = float64(usedCPUsInCluster) / float64(totalCPUsInCluster) * 100
+	}
+	fmt.Fprintf(w, "%s/%s (%d%%)\t\n",
+		resource.NewMilliQuantity(usedCPUsInCluster, resource.DecimalSI).String(),
+		resource.NewMilliQuantity(totalCPUsInCluster, resource.DecimalSI).String(),
+		int64(cpuUsage))
+
+	fmt.Fprintf(w, "Used/Total Memory In Cluster:\n")
+	log.Debugf("Memory: %s, used Memory %s", resource.NewQuantity(usedMemoryInCluster, resource.BinarySI).String(),
+		resource.NewQuantity(totalMemoryInCluster, resource.BinarySI).String())
+	var memoryUsage float64 = 0
+	if totalMemoryInCluster > 0 {
+		memoryUsage = float64(usedMemoryInCluster) / float64(totalMemoryInCluster) * 100
+	}
+	fmt.Fprintf(w, "%s/%s (%d%%)\t\n",
+		resource.NewQuantity(usedMemoryInCluster, resource.BinarySI).String(),
+		resource.NewQuantity(totalMemoryInCluster, resource.BinarySI).String(),
+		int64(memoryUsage))
 
 	_ = w.Flush()
 }
 
-func displayTopNodeDetails(nodeInfos []NodeInfo) {
+func displayTopNodeDetails(nodeInfos []NodeInfo, heapsterEndpointUrl string) {
 	w := tabwriter.NewWriter(os.Stdout, 0, 0, 2, ' ', 0)
 	var (
 		totalGPUsInCluster     int64
 		allocatedGPUsInCluster int64
+		totalCPUsInCluster     int64
+		usedCPUsInCluster      int64
+		totalMemoryInCluster   int64
+		usedMemoryInCluster    int64
 	)
 
 	fmt.Fprintf(w, "\n")
 	for _, nodeInfo := range nodeInfos {
 		totalGPU, allocatedGPU := calculateNodeGPU(nodeInfo)
+		totalCPU, usedCPU := calculateNodeCPU(heapsterEndpointUrl, nodeInfo)
```

Comment:
This will request to heapster for (node num * pod num) times. If we want to get real-time metric, maybe we can reference the Implementation of `kubectl top node / kubectl top pod`
https://github.com/kubernetes/kubernetes/blob/029bb4e213502268af6222cc2f6903ab90de751b/pkg/kubectl/metricsutil/metrics_client.go#L67

Reply:
Oh, thanks for your kind reminding, then we could just do nothing but include this `metrics_client.go` into our `commands` package~